src/share/mersenne_twister.f

```diff
@@ -173,7 +173,7 @@ module mersenne_twister
         integer,parameter:: n=624
         integer,parameter:: m=397
         integer,parameter:: mata=-1727483681 ! constant vector a
-        integer,parameter:: umask=-2147483648 ! most significant w-r bits
+        integer,parameter:: umask=-2147483647 - 1 ! most significant w-r bits
```

Comment:
Perhaps using Fortran intrinsic functions? For example: huge()

Reply:
Sure, I just updated lmask and umask to use `HUGE(0)`